Dshop/Dshop/urls.py

```diff
@@ -37,7 +37,18 @@
     path('products/', include('apps.products_catalogue.urls')),
     path('payments/', include('apps.payments.urls')),
     path('users/', include('apps.users.urls')),
-    path('tinymce/',include('tinymce.urls')),
+    path('tinymce/', include('tinymce.urls')),
+
+    # Api
```

Comment:
Please remove comment, it's not needed. Code is self-explanatory.
If you really want, you can move api urls to api-urls.py file and include here.